tutorial/Lab2/README.md

```diff
@@ -0,0 +1,155 @@
+# Lab 2. I need to change but none of the hassle
+
+In [Lab 1](../Lab1/README.md), we installed the guestbook sample app by using Helm and saw the benefits over using `kubectl`. You probably think that you're done and know enough to use Helm. But what about updates or improvements to the chart? How do you update your running app to pick up these changes? 
+
+In this lab, we're going to look at how to update our running app when the chart has been changed. To demonstrate this, we're going to make changes to the original `guestbook` chart by:
+* Removing the Redis slaves and using just the in-membory DB
+* Changing the type from `LoadBalancer` to `NodePort`.
```

Comment:
Sorry if I am missing something but Lab1 chart output shows NodePort .. so what are we changing here?

Reply:
Good catch. Updated the `NOTES.txt` and the command outputs where appropriate.